dali/operators/signal/fft/power_spectrum.h

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_OPERATORS_SIGNAL_FFT_POWER_SPECTRUM_H_
+#define DALI_OPERATORS_SIGNAL_FFT_POWER_SPECTRUM_H_
+
+#include <string>
+#include <vector>
+#include "dali/core/common.h"
+#include "dali/kernels/kernel_manager.h"
+#include "dali/kernels/signal/fft/fft_cpu.h"
+#include "dali/pipeline/operator/common.h"
+#include "dali/pipeline/operator/operator.h"
+
+namespace dali {
+
+template <typename Backend>
+class PowerSpectrum : public Operator<Backend> {
+ public:
+  explicit inline PowerSpectrum(const OpSpec &spec)
```

Comment:
```suggestion
  explicit PowerSpectrum(const OpSpec &spec)  // NOLINT
```
If method is defined in class body it's implicitly `inline`. I believe that declaration is redundant. Also linter complaints on `explicit`

Reply:
will change